### PR TITLE
genesishash and brokeraddresses real exchange

### PR DESCRIPTION
**Pull request by tenisakb**

### CODE REVIEW

File: directories/src/redux/services/brokernode.js [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+    .then(data => {
+      resolve({ data });
+    })
+    .catch(error => {
+      console.log("PUT complete " + error);
+      reject(API_REQUEST_ERROR);
+    });
+  });
+
+const broadcastToHooks = (trytes, nodes) => {
```
Comment (rfornea) [inline directories/src/redux/services/brokernode.js]: We will no longer need to broadcast to the hooknodes anymore.  For the genesis hash and brokernode address transactions, the brokernode will do the broadcast for us when we make the API call to complete the transaction.   

For the treasure hunting, I think we said we'd just use the brokernode's Iota API to do the broadcasting.  So we'll still need a method for that but it won't be calling http://${node}:3000/broadcast/

File: brokernode-api/service/BrokersService.js [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+  "pow" : {
+    "branchTransaction" : "Branch transaction example",
+    "address" : "Address example",
+    "trunkTransaction" : "Trunk transaction example",
+    "message" : "tx1"
+  },
+  "id" : true
+}, {
+  "pow" : {
+    "branchTransaction" : "Branch transaction example",
```
Comment (rfornea) [inline brokernode-api/service/BrokersService.js]: This isn't urgent but at some point we should use better examples for these values.  Such as some valid trytes values that you might see in a real exchange.

Comment (rfornea) [reply brokernode-api/service/BrokersService.js]: Never mind, disregard this comment.  Wrong file.

File: brokernode-api/api/swagger.yaml [hunk 3]
```diff
@@ -0,0 +1,303 @@
+---
+swagger: "2.0"
```
Comment (rfornea) [inline brokernode-api/api/swagger.yaml]: This is the file I meant to comment on.  At some point we should try to use realistic examples like what you might see in the real application.

File: brokernode-api/controllers/Brokers.js [hunk 4]
```diff
@@ -0,0 +1,50 @@
+'use strict';
```
Comment (EdmundMai) [inline brokernode-api/controllers/Brokers.js]: are these files generated by swagger?

File: directories/src/components/bootstrap/index.js [hunk 5]
```diff
@@ -0,0 +1,33 @@
+import React, { Component } from "react";
```
Comment (EdmundMai) [inline directories/src/components/bootstrap/index.js]: what is this component for? don't we already have one?

File: directories/src/redux/services/brokernode.js [hunk 6]
```diff
@@ -0,0 +1,107 @@
...
+      title = "brokernodes";
+      break;
+    case 1:
+      title = "genesis_hash";
+      break;
+    default:
+      title = "brokernodes";
+  }
+  return title;
+};
```
Comment (EdmundMai) [inline directories/src/redux/services/brokernode.js]: i don't think it should be random. currently there's an (simplified) epic for it in node-epics. look for `DETERMINE_REQUEST`

File: directories/src/redux/services/brokernode.js [hunk 7]
```diff
@@ -0,0 +1,107 @@
...
+
+const randomFn = (maxNumber) => {
+  const random = Math.floor(Math.random() * (maxNumber + 1));
```
Comment (EdmundMai) [inline directories/src/redux/services/brokernode.js]: `return Math.floor(Math.random() * (maxNumber + 1));`

**Comment by EdmundMai** [comment]
i don't see anything related to genesis hashes in this PR ?

File: brokernode-api/api/swagger.yaml (hunk 3 above)
Comment (EdmundMai) [reply brokernode-api/api/swagger.yaml]: yeh lets use real Tangle addresses / messages / trunk / branches

**Pull request closed without merging**